.dev/docker/windowsx64/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# escape=`
```

Comment:
Can we use build-time variables to make one dockerfile instead of these 2 dockerfiles?

Reply:
I have added an ARG to supply if it should be x86 or x64. I think it works, however I haven't completely rebuild two new images yet. Failed due to space limits :P